packages/test/test-service-load/src/runner.ts

```diff
@@ -86,11 +88,34 @@ async function runnerProcess(
             // Switch between creating new factory vs. reusing factory.
             // Certain behavior (like driver caches) are per factory instance, and by reusing it we hit those code paths
             // At the same time we want to test newly created factory.
-            const factory = counter % 1 === 0 ?
-                documentServiceFactoryReused :
-                new FaultInjectionDocumentServiceFactory(testDriver.createDocumentServiceFactory());
+            let factory: FaultInjectionDocumentServiceFactory = documentServiceFactoryReused;
+            let headers: IRequestHeader = {};
+            let resume = false;
+            switch (counter % 5) {
+                default:
+                case 0:
+                    factory = new FaultInjectionDocumentServiceFactory(testDriver.createDocumentServiceFactory());
+                    break;
+                case 1:
+                    headers = { [LoaderHeader.loadMode]: { opsBeforeReturn : "cached"} };
+                    break;
+                case 2:
+                    headers = { [LoaderHeader.loadMode]: { opsBeforeReturn : "all"} };
+                    break;
+                case 3:
+                    resume = true;
+                    headers = { [LoaderHeader.loadMode]: { deltaConnection : "none"} };
+                    break;
+                case 4:
+                    resume = true;
+                    headers = { [LoaderHeader.loadMode]: { deltaConnection : "delayed"} };
+                    break;
+                }
 
-            const {container, test} = await load(testDriver, factory, url, runConfig.runId);
+            const {container, test} = await load(testDriver, factory, { url, headers }, runConfig.runId);
+            if (resume) {
+                container.resume();
```

Comment:
is it safe to just always call this?

Reply:
of is there a way to know if it needs to be call from the container itself?

Reply:
Yes, it's always safe to call it (many times). I'll make it unconditional.